server/models/Profile.js

```diff
@@ -0,0 +1,21 @@
+const mongoose = require("mongoose");
+
+const profileSchema = new mongoose.Schema({
+  isDogSitter: { type: Boolean, require: true, default: false },
+  isAvailable: { type: Boolean, require: true, default: false },
+  firstName: { type: String, require: true },
+  lastName: { type: String, require: true },
+  gender: String,
+  birthDate: Date,
+  email: String,
+  phoneNumber: String,
+  address: String,
+  description: String,
+  profileImg: String,
+  coverImg: String,
+  galleryImg: [String],
+  availability: [String],
```

Comment:
we can discuss tomorrow in standup how we will handle the profile schema since Abhi is also working on it and @joelmackenz is also supposed to work on availability

Reply:
worked with joel, and added all fields are needed.